Meta/Lagom/Tools/LibJSGCVerifier/.gitignore

```diff
@@ -0,0 +1,3 @@
+build/
```

Comment:
From the PR description: 
> [This error](https://i.imgur.com/n7TOZeK.png), not sure why it happens or what to do about it

This looks like has to do with include path handling in the main build. If you insert an include path that has standard libC header files *before* the compiler default include paths in the search list, all the `#include_next` behavior gets super messed up in clang/libc++'s builtin header files that patch over LibC header files to make them wrappable by the C++ STL. We talked about this on... the syzcaller issue iirc.

Reply:
Yeah I don't think I'm knowledgeable enough to fix this :) at the end of the day it's a few line of error output, and I'd much rather spend my time using this tool to start the conversion rather than focus on this